<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KJobCreator</class>
 <widget class="QDialog" name="KJobCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KJobCreator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Desktop Entry</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="desktopEntry"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DestUrl</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="destUrl"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="descriptionLabel1">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Label 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="descriptionValue1">
       <property name="placeholderText">
        <string>Description 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="descriptionLabel2">
       <property name="placeholderText">
        <string>Label 2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="descriptionValue2">
       <property name="placeholderText">
        <string>Description 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="emitDescriptionButton">
         <property name="text">
          <string>Emit &amp;Description</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Percent</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSlider" name="percent">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bytes</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="processedBytes">
         <property name="suffix">
          <string> MiB</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bytesOfLabel">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="totalBytes">
         <property name="suffix">
          <string> MiB</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="speed">
       <property name="suffix">
        <string> MiB/s</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Info Message</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="infoMessage"/>
       </item>
       <item>
        <widget class="QPushButton" name="emitInfoMessage">
         <property name="text">
          <string>Emit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QComboBox" name="errorCombo">
         <item>
          <property name="text">
           <string>No Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Canceled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="errorText">
         <property name="placeholderText">
          <string>Error message</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="title">
       <property name="text">
        <string>Test Job</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="processedFiles"/>
       </item>
       <item>
        <widget class="QLabel" name="filesOfLabel">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="totalFiles"/>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QCheckBox" name="suspendableCheck">
         <property name="text">
          <string>Suspendable</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="killableCheck">
         <property name="text">
          <string>Killable</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="immediateCheck">
         <property name="text">
          <string>Immediate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="transientCheck">
         <property name="text">
          <string>Transient</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Directories</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QSpinBox" name="processedDirectories"/>
       </item>
       <item>
        <widget class="QLabel" name="directoriesOfLabel">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="totalDirectories"/>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QSpinBox" name="processedItems"/>
       </item>
       <item>
        <widget class="QLabel" name="itemsOfLabel">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="totalItems"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Register with</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerKWidgetJobTracker">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>KWidgetJobTracker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerKUIServerJobTracker">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>KUIServerJobTracker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerKUIServerV2JobTracker">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>KUIServerV2JobTracke&amp;r</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>&amp;Create</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="suspendButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sus&amp;pend</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="killButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Kill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
